libindy/src/services/wallet/mod.rs

```diff
@@ -519,13 +532,76 @@ impl WalletService {
         }
     }
 
+    pub fn export_wallet(&self, wallet_handle: i32, export_config_json: &str, version: u32) -> Result<(), WalletError> {
+        match self.wallets.borrow().get(&wallet_handle) {
+            Some(wallet) => {
+                let export_config: WalletExportConfig = serde_json::from_str(export_config_json)?;
+                let export_path = Path::new(&export_config.path);
+                if export_path.exists() {
+                    return Err(WalletError::ExportPathExists);
+                }
+
+                let export_file = File::create(export_path)?;
```

Comment:
use https://doc.rust-lang.org/std/fs/struct.OpenOptions.html instead of manual checking